org_status/org_hosts/gitlab.py

```diff
@@ -39,8 +43,19 @@ def process_repository(self, repo, branch='master'):
                                                        self.HostName,
                                                        branch=branch)
 
-        return RepoStatus(repo.web_url, repo_status)
+        last_commit_date = repo.get_last_commit_date()
+
+        return RepoStatus(repo.web_url, repo_status, last_commit_date)
 
     @property
     def repositories(self):
         return self._org.repositories
+
+
+def _get_last_commit_date(self):
+    repo_commit_dates = ['']
+    for commit in self.commits:
```

Comment:
So, this will not work with `--use-repo-list` mode from https://github.com/ksdme/org-status/pull/34, right?

Reply:
Why iterate over every commit?

That is horribly inefficient.

Any list of commits should be already pre-sorted, most recent first, so you should be able to get only the first using `next(iter(commit_list))`

Reply:
Yes, the list of commits is not pre-sorted.

Reply:
Ok, we'll get IGitt fixed so this doesnt need to be so ugly.